ruby_on_rails/app_initialisation.md

```diff
@@ -58,7 +58,7 @@ They are always idempotent (runnable multiple times).
 * Add a `bin/check` file. It will run all the automated tests. It's mainly used in our CI.
 
   ```sh
-  echo "#\!/usr/bin/env bash\nset -euo pipefail\nbin/fastcheck\nbin/rails zeitwerk:check" > bin/check
+  echo "#\!/usr/bin/env bash\nset -euo pipefail\n\nbin/rails zeitwerk:check" > bin/check
```

Comment:
I think a double new line increases readability and we also have it this way in other projects as well (e.g. mtextur, remms)